src/definitions.ts

```diff
@@ -121,23 +121,31 @@ export interface Location {
   latitude: number;
   longitude: number;
   accuracy?: number;
+  speed?: number;
+  altitude?: number;
+  course?: number;
+  verticalAccuracy?: number;
+  speedAccuracy?: number;
+  courseAccuracy?: number;
 }
 
 export interface RadarUser {
   _id: string;
   userId?: string;
   deviceId?: string;
   description?: string;
+  stopped?: boolean;
+  segments?: RadarSegment[];
```

Comment:
Should we add the rest? Glancing at a response:
```
debug?: boolean,
updatedAt: Date,
deviceId?: string,
installId?: string,
sdkVersion?: string,
deviceOS?: string,
deviceMake?: string,
deviceModel?: string,
userAgent?: string,
sessionId?: string,
location?: {
type: "Point",
coordinates: [lng, lat]
}
```

Reply:
i dont think many of these are in the plugin mapping: https://github.com/radarlabs/radar-sdk-ios/blob/7769694225561ec697ccf8c9869a9b28d42171e0/RadarSDK/RadarUser.m#L298

Reply:
can keep to Point for now. we only seen to change to flatten the location in top level location objects.